File: android/app/src/main/java/com/appxstudios/festivalconnection/ui/screens/LaunchScreen.kt
package com.appxstudios.festivalconnection.ui.screens

import androidx.compose.foundation.Image
import androidx.compose.foundation.background
import androidx.compose.foundation.layout.*
import androidx.compose.foundation.shape.RoundedCornerShape
import androidx.compose.material3.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.Alignment
import androidx.compose.ui.Modifier
import androidx.compose.ui.draw.clip
import androidx.compose.ui.graphics.Color
import androidx.compose.ui.layout.ContentScale
import androidx.compose.ui.res.painterResource
import androidx.compose.ui.text.font.FontWeight
import androidx.compose.ui.unit.dp
import androidx.compose.ui.unit.sp
import com.appxstudios.festivalconnection.R
import com.appxstudios.festivalconnection.ui.theme.*

@Composable
fun LaunchScreen() {
    Box(
        modifier = Modifier
            .fillMaxSize()
            .background(BackgroundBlack),
        contentAlignment = Alignment.Center
    ) {
        Column(
            horizontalAlignment = Alignment.CenterHorizontally,
            verticalArrangement = Arrangement.Center,
            modifier = Modifier.fillMaxSize()
        ) {
            Spacer(Modifier.weight(1f))

            // Title
            GradientText(
                text = "Festival Connection",
                fontSize = 32.sp,
                fontWeight = FontWeight.ExtraBold
            )

            Spacer(Modifier.height(40.dp))

            // App icon
            Image(
                painter = painterResource(R.drawable.launch_icon),
                contentDescription = "Festival Connection",
                modifier = Modifier
                    .size(200.dp)
                    .clip(RoundedCornerShape(40.dp)),
                contentScale = ContentScale.Crop
            )

            Spacer(Modifier.weight(1f))

            // Powered by
            Column(
                horizontalAlignment = Alignment.CenterHorizontally,
                modifier = Modifier.padding(bottom = 60.dp)
            ) {
                Text(
                    "Powered by",
                    fontSize = 13.sp,
                    color = TextMuted
                )
                Spacer(Modifier.height(4.dp))
                GradientText(
                    text = "CrowdSync\u2122",
                    fontSize = 17.sp,
                    fontWeight = FontWeight.SemiBold
                )
            }
        }
    }
}

File: ios/FestivalConnection/UI/Screens/LaunchScreen.swift
import SwiftUI

struct LaunchScreen: View {
    var body: some View {
        ZStack {
            FestivalTheme.backgroundBlack.ignoresSafeArea()

            VStack(spacing: 0) {
                Spacer()

                // Title
                Text("Festival Connection")
                    .font(.system(size: 32, weight: .heavy))
                    .foregroundStyle(FestivalTheme.mainGradient)
                    .padding(.bottom, 40)

                // App icon
                Image("LaunchIcon")
                    .resizable()
                    .aspectRatio(contentMode: .fit)
                    .frame(width: 200, height: 200)
                    .clipShape(RoundedRectangle(cornerRadius: 40))

                Spacer()

                // Powered by
                VStack(spacing: 4) {
                    Text("Powered by")
                        .font(.system(size: 13, weight: .regular))
                        .foregroundColor(FestivalTheme.textMuted)
                    Text("CrowdSync\u{2122}")
                        .font(.system(size: 17, weight: .semibold))
                        .foregroundStyle(FestivalTheme.mainGradient)
                }
                .padding(.bottom, 60)
            }
        }
    }
}
